sys/linux/devio.txt

```diff
@@ -0,0 +1,129 @@
+# Copyright 2019 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# https://elixir.bootlin.com/linux/latest/source/drivers/usb/core/devio.c
+
+include <linux/ioctl.h>
+include <linux/usbdevice_fs.h>
+include <uapi/linux/fcntl.h>
+
+define MAXDRIVERNAME	USBDEVFS_MAXDRIVERNAME + 1
+
+resource fd_devio[fd]
```

Comment:
Let's name this one `fd_usbfs` and name all of the related syscalls `*$usbfs` accordingly.